components/automate-ui/src/app/modules/infra-proxy/create-chef-server-modal/create-chef-server-modal.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Input, Output, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-create-chef-server-modal',
+  templateUrl: './create-chef-server-modal.component.html',
+  styleUrls: ['./create-chef-server-modal.component.scss']
+})
+export class CreateChefServerModalComponent implements OnInit {
```

Comment:
_Comment for future work--not for now:_

This code follows our existing patterns fine. But we recently introduced a new, cleaner pattern that I would like you and your team to be aware of for modals--see https://github.com/chef/automate/pull/2704. It encapsulates the code for the modal in a much cleaner way.

Again, no need to apply to this PR; sounds like we want to get this merged ASAP. But please take a look in case of future modals.